### PR TITLE
Internal api doc

### PR DESCRIPTION
**Pull request by aharin90**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,3 +303,57 @@ The `./sdk-generation` directory contains code and config for our SDK generation
...
+    ...
+    <Contact internal=`true`> <-- contains marker
+      <FirstName>Sanjay</FirstName>
+      <LastName>Rao</LastName>
+      <Phone>9195441234</Phone>
+      <Email>srao@bandwidth.com</Email>
+    </Contact>
+    <AltSpid>X455</AltSpid>
+    <SPID>9999</SPID>
+    <PortCarrierType internal=`true`>WIRELINE</PortCarrierType> <--contains marker
```
Comment (ajrice6713) [inline README.md:339]: AFAIK openapi doesnt support XML with both elements and attributes 

Does this processor read/write the xml values from/to the example field? Or the model schema?

File: README.md [hunk 2]
```diff
@@ -303,3 +303,57 @@ The `./sdk-generation` directory contains code and config for our SDK generation
 | php | PHP >= 7.2
 | phpold | Does not have XML support |
 | node | Highly recommended to generate with a specified spec |
+
+
+## Api Doc preprocessor
```
[the remark this answers is not in the document]
Comment (aharin90) [reply README.md:308]: Hey @ajrice6713! Thanks for replying. I agree with you, this jar would be used in future by GHA. It is very first step for API doc preprocession. I made updates in README just for acquaintance with such opportunity. I think for now we can keep it in separate branch.

[the remark this answers is not in the document]
Comment (ajrice6713) [reply README.md:308]: Will mark as draft PR then - Thanks @aharin90

**Comment by mrmeyers99** [comment]
Can you put the actual code here instead of the Jar file?  We'll probably build it during the GHA

**Pull request closed without merging**